frontend/packages/console-app/src/components/nodes/NodeDetailsOverview.tsx

```diff
@@ -92,7 +92,7 @@ const NodeDetailsOverview: React.FC<NodeDetailsOverviewProps> = ({ node }) => {
               canEdit={canUpdate}
               editAsGroup
             >
-              <LabelList kind="Node" labels={node.metadata.labels} />
+              <LabelList kind="Node" labels={node.metadata?.labels || {}} />
```

Comment:
maybe we can make `labels` an optional prop here as well and set `{}` as the default value